packages/ihe-gateway-sdk/src/common/__tests__/normalize-oid.test.ts

```diff
@@ -0,0 +1,26 @@
+import { normalizeOid } from "../shared";
+const validOid = "1.22.333.444";
+const validOidWithPrefix = "urn:oid:1.22.333.444";
```

Comment:
NABD, but its a good idea to test multiple inputs in cases like this. For example:
- does an one-level OID work?
- does an OID with 10 levels/periods work?

Also, could test with OID with to consecutive periods (`1.2..3`), period at the end and at the end